velodyne_pointcloud/include/velodyne_pointcloud/point_types.h

```diff
@@ -19,28 +19,39 @@
 #ifndef __VELODYNE_POINTCLOUD_POINT_TYPES_H
 #define __VELODYNE_POINTCLOUD_POINT_TYPES_H
 
-#include <pcl/point_types.h>
+#include <sensor_msgs/PointCloud2.h>
+#include <sensor_msgs/point_cloud2_iterator.h>
 
 namespace velodyne_pointcloud
 {
   /** Euclidean Velodyne coordinate, including intensity and ring number. */
-  struct PointXYZIR
+/*  struct PointXYZIR
```

Comment:
The driver defines a custom point type, so it can publish the laser ring number with the X, Y, Z and intensity of each point.

Getting rid of that is definitely an API breakage.